package/boot/uboot-mvebu/patches/120-mvebu_armada-37xx.h_increase_max_gunzip_size.patch

```diff
@@ -0,0 +1,10 @@
+--- a/include/configs/mvebu_armada-37xx.h
```

Comment:
Would be nice to have here the same description as in commit message.

Reply:
Added info in the patch itself.